Client/Coordinators/TabTrayCoordinator.swift

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+
+protocol TabTrayCoordinatorDelegate: AnyObject {
+    func didDismissTabTray(from coordinator: TabTrayCoordinator)
+}
+
+class TabTrayCoordinator: BaseCoordinator {
```

Comment:
We can create a unit test file already for that coordinator. We can check that the root view controller is set properly I think